Dockerfile.ocp

```diff
@@ -17,6 +17,6 @@ LABEL io.k8s.display-name="kube-rbac-proxy" \
 ARG FROM_DIRECTORY=/go/src/github.com/brancz/kube-rbac-proxy
 COPY --from=builder ${FROM_DIRECTORY}/_output/kube-rbac-proxy  /usr/bin/kube-rbac-proxy
 
-USER nobody
+USER 65534
```

Comment:
So we just want this to be nobody but numericly? 

In CMO we use another user UID, hence the question.

Reply:
correct, we still want this to be nobody, but numeric. the whole problem is that if we use the literal string `"nobody"` non-numerically, along with the `nonroot` security context constraint, the provisioning will fail with the following error:
```
container has runAsNonRoot and image has non-numeric user (nobody), cannot verify user is non-root
```

Reply:
and we tried removing the `nonroot` security context constraint from CMO alltogether, but that didn't work of the following provisioning error: https://github.com/openshift/cluster-monitoring-operator/pull/1015#issuecomment-742416683

Reply:
Thanks for the explanation!!

/hold cancel

Reply:
side note: i am not very happen about the solution to be honest, the question around the mystery why cluster-monitoring-operator _sometimes_ gets `nonroot` assigned vs. `restricted` still remains (hypothesis so far is some race in the provisioning controller). Additionally, i do get the necessity for cluster-monitoring-operator needs to have the same security context constraint, but it is not really necessary for the pod runtime as we don't mount things there.